src/pages/K8s.vue

```diff
@@ -225,6 +249,21 @@ export default {
         }
       }
     },
+    async onChangeEngine(desiredEngine) {
+      if (desiredEngine !== this.settings.kubernetes.containerEngine) {
+        const confirmationMessage = [`Changing container engines from ${ this.containerEngineNames[this.currentEngine] } to ${ this.containerEngineNames[desiredEngine] } will require a full reset of Kubernetes (loss of workloads) )`,
```

Comment:
Is it a "full" reset? That sounds like a wipe.

Reply:
Good catch -- it's just a restart, and images are (with this PR) preserved.